applications/CompressiblePotentialFlowApplication/python_scripts/define_wake_process_2d.py

```diff
@@ -64,8 +47,12 @@ def __init__(self, Model, settings):
         for cond in self.body_model_part.Conditions:
             for node in cond.GetNodes():
                 node.Set(KratosMultiphysics.SOLID)
-
+    
     def ExecuteInitialize(self):
+        self.FindWakeElements()
+
+    def FindWakeElements(self):
```

Comment:
Taking into account that this method doesnt't match the `Process`API, it assume that its usage is private in here. If this is the case, please follow the convention an name it using `snake_case` instead of `CamelCase`.

Reply:
(as well as the rest of private methods)

Reply:
actually private methods are `__PrivateMethod` and protected methods are `_ProtectedMethods`

This has been discussed long ago and also it is part of the style guide, but so far we have not strongly enforced it

Reply:
Ok!